Tests/swift-sdk-swift-tests/IterableActionRunnerTests.swift

```diff
@@ -27,12 +26,16 @@ class IterableActionRunnerTests: XCTestCase {
     
     func testUrlOpenAction() {
         let urlString = "https://example.com"
-        let urlDelegate = MockUrlDelegate(returnValue: false)
+        let action = IterableAction.action(fromDictionary: ["type" : "openUrl", "data" : urlString])!
+        let context = IterableActionContext(action: action, source: .push)
         let urlOpener = MockUrlOpener()
-        let actionRunner = IterableActionRunner(urlDelegate: urlDelegate, customActionDelegate: nil, urlOpener: urlOpener)
+
+        let handled = IterableActionRunner.execute(action: action,
```

Comment:
I think this should also verify that urlHandler was called, similar to `testCustomAction` below.